share/ruby-build/truffleruby-23.0.0

```diff
@@ -0,0 +1,21 @@
+platform="$(uname -s)-$(uname -m)"
+case $platform in
+Linux-x86_64)
+  install_package "truffleruby-23.0.0" "https://gds.oracle.com/api/20220101/artifacts/FD4AB182EA4CEDFDE0531518000AF13E/content#57f770e7ba754327c8c160571978c7da0e10a086f8b5dc542e5c8ff5b2c83190" truffleruby
```

Comment:
These URLs are not pretty, but they are stable and should keep working forever.
We're working on getting a stable URL format for them, similar to `truffleruby+graalvm-23.0.0` in the other file.